### ISSUE TITLE
`Maybe (List Record)` can't be expanded

### ISSUE THREAD

**Issue by Janiczek** [kraklin/elm-debug-transformer#6]
![debug](https://user-images.githubusercontent.com/149425/60652112-dc748080-9e47-11e9-9cb4-48da2525668c.png)

I see that it contains some data, but I can't drill down into that data.

The log message in question is:
```elm
maybe query: Just [{ audienceId = "0", channelId = "68325", data = Ok { averageFrequency = 1, grp = 2.529294547581681, impacts = 966815, reach = 966815, reachPercentage = 2.5292945475816806, sampleSize = 34325, universe = 38224690 }, locationId = "", requestId = "a777f082-a27e-410a-affc-5104addddf70" }]
```

version: `elm-debug-transformer@1.0.0`

**Comment by kraklin** [comment]
It should be solved with version `elm-debug-transformer@1.0.0-rc2`. Can you try it?

**Comment by Janiczek** [comment]
![transform](https://user-images.githubusercontent.com/149425/61305174-a55c8280-a7ea-11e9-8aaf-0668dc030ad1.png)
Yup, solved!
